src/main/java/dinkplugin/notifiers/data/LevelNotificationData.java

```diff
@@ -1,11 +1,30 @@
 package dinkplugin.notifiers.data;
 
+import dinkplugin.message.Field;
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
 @Value
-public class LevelNotificationData {
+@EqualsAndHashCode(callSuper = false)
+public class LevelNotificationData extends NotificationData {
     Map<String, Integer> levelledSkills;
     Map<String, Integer> allSkills;
+
+    @Override
+    public List<Field> getFields() {
```

Comment:
I don't like this field since it looks through all your skills, so if a user levels their agility to 83 but has 103 attack & defense, this will say "Total 99+ skills: 2" which seems irrelevant

Reply:
It only adds the 99+ field if the *leveled* skill has maxed (e.g., agility hit 99, indicate that user has 3 99's now)

Reply:
Ah yeah, the L19 if handles that.
I'm still not sold on the field personally. Would be nice with someone else's opinion too

Reply:
I can take it or leave it, I understand the reasoning but it probably makes no sense for it to re-fire on virtual levels.

Reply:
compromise: remove it for now and add in a separate PR to have the discussion there instead?